screen-weather-get.py

```diff
@@ -159,21 +168,33 @@ def main():
     alert_message = get_alert_message(location_lat, location_long)
     alert_message = format_alert_description(alert_message)
 
+    try:
+        time_now = format_time(datetime.datetime.now(), format='short', locale=locale.getlocale()[0])
+    except Exception:
+        logging.debug("Locale not found for Babel library.")
+        time_now = datetime.datetime.now().strftime("%-I:%M %p")
+
+    time_now_font_size = "100px"
+
+    if len(time_now) > 6:
+        time_now_font_size = str(100 - (len(time_now)-5) * 5) + "px"
```

Comment:
Unfortunately no way in SVG to tell some text to fill up and stretch to its container.  Instead I have to reduce the font size if the time text is larger. For example, "20:54" can be 100px, but "12:33 PM" would be smaller.  

- [ ] Still, need to check if the other layouts can work with this.